bin/pact-cli.ts

```diff
@@ -26,6 +26,18 @@ cli
 	.option("--provider [providerName]", "Specify provider name for written Pact files.")
 	.action((args: any, options: any) => pact.createServer(options).start());
 
+cli
+	.command("stub")
+	.description("Creates an API stub from pact files")
+	.option("-p, --port [n]", "Port on which to run the service. Default is random.", cli.INT)
```

Comment:
Please change all square brackets for angle brackets as per my new feature breach.

Reply:
Done.